src/platform/macos.rs

```diff
@@ -226,3 +228,264 @@ impl<T> EventLoopWindowTargetExtMacOS for EventLoopWindowTarget<T> {
+            KeyCode::F9 => Some(0x65),
+            KeyCode::F11 => Some(0x67),
+            KeyCode::F13 => Some(0x69),
+            KeyCode::F16 => Some(0x6a),
+            KeyCode::F14 => Some(0x6b),
+            KeyCode::F10 => Some(0x6d),
+            KeyCode::F12 => Some(0x6f),
+            KeyCode::F15 => Some(0x71),
+            KeyCode::Insert => Some(0x72),
+            KeyCode::Home => Some(0x73),
+            KeyCode::PageUp => Some(0x74),
+            KeyCode::Delete => Some(0x75),
+            KeyCode::F4 => Some(0x76),
+            KeyCode::End => Some(0x77),
+            KeyCode::F2 => Some(0x78),
+            KeyCode::PageDown => Some(0x79),
+            KeyCode::F1 => Some(0x7a),
+            KeyCode::ArrowLeft => Some(0x7b),
+            KeyCode::ArrowRight => Some(0x7c),
+            KeyCode::ArrowDown => Some(0x7d),
+            KeyCode::ArrowUp => Some(0x7e),
+            _ => None,
+        }
+    }
+
+    fn from_scancode(scancode: u32) -> KeyCode {
+        match scancode {
+            0x00 => KeyCode::KeyA,
+            0x01 => KeyCode::KeyS,
+            0x02 => KeyCode::KeyD,
+            0x03 => KeyCode::KeyF,
+            0x04 => KeyCode::KeyH,
+            0x05 => KeyCode::KeyG,
+            0x06 => KeyCode::KeyZ,
+            0x07 => KeyCode::KeyX,
+            0x08 => KeyCode::KeyC,
+            0x09 => KeyCode::KeyV,
+            //0x0a => World 1,
+            0x0b => KeyCode::KeyB,
+            0x0c => KeyCode::KeyQ,
+            0x0d => KeyCode::KeyW,
+            0x0e => KeyCode::KeyE,
+            0x0f => KeyCode::KeyR,
+            0x10 => KeyCode::KeyY,
+            0x11 => KeyCode::KeyT,
+            0x12 => KeyCode::Digit1,
+            0x13 => KeyCode::Digit2,
+            0x14 => KeyCode::Digit3,
+            0x15 => KeyCode::Digit4,
+            0x16 => KeyCode::Digit6,
+            0x17 => KeyCode::Digit5,
+            0x18 => KeyCode::Equal,
+            0x19 => KeyCode::Digit9,
+            0x1a => KeyCode::Digit7,
+            0x1b => KeyCode::Minus,
+            0x1c => KeyCode::Digit8,
+            0x1d => KeyCode::Digit0,
+            0x1e => KeyCode::BracketRight,
+            0x1f => KeyCode::KeyO,
+            0x20 => KeyCode::KeyU,
+            0x21 => KeyCode::BracketLeft,
+            0x22 => KeyCode::KeyI,
+            0x23 => KeyCode::KeyP,
+            0x24 => KeyCode::Enter,
+            0x25 => KeyCode::KeyL,
+            0x26 => KeyCode::KeyJ,
+            0x27 => KeyCode::Quote,
+            0x28 => KeyCode::KeyK,
+            0x29 => KeyCode::Semicolon,
+            0x2a => KeyCode::Backslash,
+            0x2b => KeyCode::Comma,
+            0x2c => KeyCode::Slash,
+            0x2d => KeyCode::KeyN,
+            0x2e => KeyCode::KeyM,
+            0x2f => KeyCode::Period,
+            0x30 => KeyCode::Tab,
+            0x31 => KeyCode::Space,
+            0x32 => KeyCode::Backquote,
+            0x33 => KeyCode::Backspace,
+            //0x34 => unkown,
```

Comment:
nit: `unknown`

Reply:
Thanks for taking a look, and for bringing this to my attention. 😊 Should be correct now.